jannovar-cli/src/main/java/de/charite/compbio/jannovar/cmd/JannovarAnnotationCommand.java

```diff
@@ -21,7 +21,7 @@
 	/** Map of Chromosomes, used in the annotation. */
 	protected ImmutableMap<Integer, Chromosome> chromosomeMap = null;
 	
-	protected JannovarData data = null;
+	protected  JannovarData data = null;
```

Comment:
Your space is wrong. So look up the changes before are use git commit.
And do not use auto formating.